### PR TITLE
[FLINK-30352] [connectors/elasticsearch] Document missing configuration properties

### PR DESCRIPTION
**Pull request by reta**
There is a number of configuration properties which are not documented:
- sink.delivery-guarantee
- connection.request-timeout
- connection.timeout
- socket.timeout

### CODE REVIEW

**Comment by reta** [comment]
@zentol please take a look, closing the gap in Elasticsearch configuration documentation, thank you

**Review by zentol — COMMENTED** [review]
Please also update the chinese version of the docs under content.zh.

**Comment by zentol** [comment]
> @zentol what is the process here? I can only do English, someone will translate?

You can add the snippets in english, and someone may pick up the translation at some point.

File: docs/content.zh/docs/connectors/table/elasticsearch.md [hunk 1]
```diff
@@ -143,6 +143,19 @@ CREATE TABLE myUserTable (
       </ul>
       </td>
     </tr>
+    <tr>
```
Comment (reta) [inline docs/content.zh/docs/connectors/table/elasticsearch.md:146]: The `timeout`s were documented, only `sink.delivery-guarantee` were missed

**Pull request merged**